apps/server/src/modules/learnroom/common-cartridge/common-cartridge.enums.ts

```diff
@@ -1,13 +1,15 @@
 export enum CommonCartridgeVersion {
+	V_1_0_0 = '1.0.0',
```

Comment:
Should we support all of these versions of CC?

Reply:
@Fshmit These are all the versions that exist. And they are listed here for completeness and in case we implement them in the future. Then we just have to add the new version in 'check supportedVersions'.